.gitignore

```diff
@@ -46,3 +46,8 @@ finetuning_ckpts/
 assertion.log
 *.log
 *.db
+.venv
+temp_req.py
```

Comment:
can you remove these extra files here? we don't want to include extraneous files in the library's gitignore